docs/getting-started/install-kubernetes.md

```diff
@@ -153,6 +153,15 @@ $ kubectl gadget deploy --apparmor-profile 'localhost/inspektor-gadget-profile'
 
 Note that, the AppArmor profile should already exists in the cluster to be used.
 
+### Deploying with a seccomp profile
+
+By default, Inspektor Gadget syscalls are not restricted.
+It is nonetheless possible to deploy Inspketor Gadget using a custom seccomp profile with `--seccomp-profile` flag:
+
+```bash
+$ kubectl gadget deploy --seccomp-profile 'gadget-profile.yaml'
+```
+
```

Comment:
Should it mention that the seccomp profile operator must be installed and that the format of the seccomp profile must be the one used in https://github.com/kubernetes-sigs/security-profiles-operator/blob/main/installation-usage.md#create-a-seccomp-profile ?